addon/templates/components/frost-list-item-content.hbs

```diff
@@ -0,0 +1,67 @@
+<div class={{concat
```

Comment:
After you clean up your cp (itemExpanded itemComponent), this template should be basically identical to the previous version (the source you're copying from) except for the unexpandable item placeholder.

Reply:
Yes after the clean up it looks almost the same as it did before with only minor changes.